src/main/resources/templates/home-page.html

```diff
@@ -0,0 +1,208 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Homepage</title>
+  <link rel="stylesheet" href="/css/styles.css">
+</head>
+<body>
+
+<header>
+    <nav>
+      <a href="/home">
+        <img src="images/confido-logo.png" style="width: 200px;padding: 10px;">
+      </a>
+      <a th:if="${electionName}" href="/view-election-details">Election Details</a>
+      <a th:if="${electionName}" href="/ledger">Ledger</a>
+      <a th:if="${electionName}" href="/">Election Results (update link!)</a>
+    </nav>
+</header>
+
+<section id="about" class="section" style="display: flex; justify-content: center; align-items: center; text-align: left; margin-top: 110px; padding: 40px; background: #FFFFFF; border-radius: 10px; max-width: 1200px; margin-left: auto; margin-right: auto; box-shadow: 0 4px 10px rgba(0, 0, 0, 0.1);">
+  <div style="flex: 1; padding: 20px;">
+    <p style="font-size: 40px; font-weight: bold; margin-bottom: 20px;">
+      Vote confidently from ANYWHERE, ANYPLACE, at ANYTIME
+    </p>
+    <p style="font-size: 20px; line-height: 1.6; margin-bottom: 20px;">
+      A redefined online voting platform that guarantees voter anonymity while maintaining a publicly available immutable voting ledger
+    </p>
+    <div class="legend-buttons" style="margin-top: 20px;">
+      <a href="#purpose" class="btn-secondary">Purpose</a>
+      <a href="#goals" class="btn-secondary">Goals</a>
+      <a href="#solutions" class="btn-secondary">Solutions</a>
+      <a href="#started" class="btn-secondary">Getting Started</a>
+    </div>
+  </div>
+  <div style="flex: 1; text-align: center;">
+    <img src="/images/map.png" style="max-width: 100%; height: auto; border-radius: 10px;">
+  </div>
+</section>
+
+
+<section id="purpose" class="section" style="text-align: center;">
+  <h2>Our Purpose</h2>
+  <p style="font-size: 20px; max-width: 800px; margin: 0 auto; margin-bottom: 20px;">
+    Your vote matters! Traditional voting systems often come with barriers that can discourage participation, making it harder for people to have their voices heard.
+  </p>
```

Comment:
```suggestion
  <p style="font-size: 20px; max-width: 800px; margin: 0 auto; margin-bottom: 20px;">
    Your vote matters! Traditional voting systems often come with barriers that <br/>
    can discourage participation, making it harder for people to have their voices heard.
  </p>
```
Improving alignment:
![image](https://github.com/user-attachments/assets/27379451-1945-4ec8-b044-5a738ee860f7)

Would be good to do the same for that purpose description just below as well.